### DISCUSSION TITLE
2023 Season Release

### DISCUSSION THREAD

**Discussion by fuddster** [PWNAGERobotics/ScoutingPASS#51] [Announcements]
2023 Updates
Scouting PASS continues to evolve. Here are the changes for the 2023 Season:

New Timer component - Start/Stop and Clear Buttons
JSON simplification - The JSON structure has been simplified to make it a little easier to maintain. The previous years configuration files have been updated to use the new structure.
Add "Flip Image" button to the Field Image component
Add "tooltip" option for components - Tooltips will appear when hovering over the name of the component
Bug Fix: Two field image elements on the same page will now render correctly
Google Sheets data export (See the [Google Sheets](https://github.com/PWNAGERobotics/ScoutingPASS/blob/main/GoogleSheets.md) page for more details)
Configurable checkbox values in configuration file. Defaults to Y/N. Can set to T/F or 1/0. See line 5 of 2022/RR_config.js for an example

<hr /><em>This discussion was created from the release <a href='https://github.com/PWNAGERobotics/ScoutingPASS/releases/tag/v2023.0'>2023 Season Release</a>.</em>

**Comment by sschrisman** [comment]
Thought would let you know that Google Sheets doesn't work.  AppScripts do not appear to be adding row to Sheet.

**Reply by fuddster** [reply]
Apologies.  We recently discovered a bug in the instructions (specifically the App script itself) that caused the problem.

Please see the instructions and copy the new code and it should work.  There is a section that shows the bad line of code and how to fix it if you just want to make that fix.

Thanks for trying ScoutingPASS!
